src/agents/aliases.ts

```diff
@@ -10,16 +10,18 @@ export const SET_ALIAS_TYPES = {
 };
 export default class Aliases extends Agent {
   async setAlias(from: string, alias: string, salt: bigint, signature: string) {
-    const recoveredAddress = await verifySignature(
+    const isSignatureValid = await verifySignature(
+      from,
       SET_ALIAS_TYPES,
       {
         from,
         alias,
         salt
       },
-      signature
+      signature,
+      { eip712: true, eip1271: true }
```

Comment:
Not sure we need these options, is there some scenario where we need to customize this?

Reply:
We currently have limitation of disallowing Safe wallets from following spaces (which seems to be UI only), I was wondering if we need to handle that case there to make it actually enforceable. If we want to go away from this limitation and we can accept its current drawbacks we can remove this "support list".